lib/sass/calcite-web.scss

```diff
@@ -30,17 +30,17 @@ $include-sticky:                 $generate-css    !default;
 
 // Type Output
 $include-type:                   $generate-css    !default;
-  $include-primary-family:       true             !default;
```

Comment:
these indentations are used to denote subgrouping - the family variables are children of that top primary variable.